lib/config/common-media-library.api.md

```diff
@@ -627,7 +627,8 @@ export function validateTracks(tracks: Track[]): TrackValidity;
 export type VideoTrack = Track & {
     width: number;
     height: number;
-    frameRate: string;
+    frameRateNum: number;
```

Comment:
I was thinking if it would be better having a particular type for this that has two attributes for instance.

Reply:
Check the new change that I made taking your comment here and let me know what do you think